cmd/traceectl/pkg/cmd/flags/policy.go

```diff
@@ -0,0 +1,12 @@
+package flags
+
+import "fmt"
+
+const PolicyFlag = "policy"
+
+func PreparePolicy(policySlice []string) ([]string, error) {
```

Comment:
This function results in the error when an empty policy input is passed (default or explicitly through `--policy \"\"`)